earthpy/tests/test_spatial.py

```diff
@@ -108,6 +109,63 @@ def test_extent_to_json():
         es.extent_to_json([0, 1, 1, 0])
 
 
+def test_normalized_diff_shapes():
+    """If provided with arrays of different shapes,
+    normalized_diff returns ValueError."""
+
+    # Test data
+    b1_2d = np.array([[6, 7, 8, 9, 10], [16, 17, 18, 19, 20]])
+    b2_1d = np.array([[1, 2, 3, 4, 5, 14, 12, 13, 14, 17]])
+
+    # Check ValueError
+    with pytest.raises(ValueError, match=r"Both arrays should be of the same dimensions"):
+        es.normalized_diff(b1 = b1_2d, b2 = b2_1d)
+
+
+def test_normalized_diff_no_mask():
```

Comment:
```
 def test_normalized_diff_no_mask(b1_b2_arrs):
     """If result of normalized_diff does not include nan values,
     array is returned as unmasked."""
 
     b1, b2 = b1_b2_arrs
     n_diff = es.normalized_diff(b1=b1, b2=b2)
 
     # Output array unmasked 
     assert ma.is_masked(n_diff) not True
```
Isn't it not the best idea to use `==` with booleans but rather just assert as assert will fail if it's false. The above should work :) (i think)

Reply:
Great point. I have replaced `== False` with `assert not`.

Reply:
yes after i wrote that i realized `assert not` is the simplest option!! great catch :)